app/models/course.rb

```diff
@@ -10,6 +10,13 @@ class Course < ApplicationRecord
 
   A_LEVEL_ATTRIBUTES = %i[a_level_subject_requirements accept_pending_a_level accept_a_level_equivalency additional_a_level_equivalencies].freeze
 
+  TRAINING_ROUTE_MAP = {
+    %w[postgraduate fee] => 'fee_funded_initial_teacher_training',
+    %w[postgraduate salary] => 'school_direct_salaried',
+    %w[postgraduate apprenticeship] => 'postgraduate_teacher_apprenticeship',
+    %w[undergraduate salary] => 'teacher_degree_apprenticeship'
```

Comment:
```suggestion
    %w[undergraduate apprenticeship] => 'teacher_degree_apprenticeship'
```
I'd have assumed this should be an apprenticeship rather than salary 🤔

Reply:
Both of these options would uniquely map to TDA but I think you're right, Apprenticeship does feel more correct.

I was previously going directly off of this mapping: https://becoming-a-teacher.design-history.education.gov.uk/becoming-a-teacher/routes-into-teaching/

but apprenticeships are salaried anyways. I've updated 👍🏼 

https://github.com/DFE-Digital/publish-teacher-training/pull/4495/commits/0b0659f1d2aee9c5c3198f2404f0696c334da741